src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -145,33 +145,74 @@ public async Task Read()
             }
         }
 
-        private static Dictionary<string, List<TimeSeriesRangeResult>> GetTimeSeriesRangeResults(DocumentsOperationContext context, string documentId, StringValues names, StringValues fromList, StringValues toList, int start, int pageSize)
+        [RavenAction("/databases/*/timeseries", "GET", AuthorizationStatus.ValidUser)]
```

Comment:
I think this requires studio change as well
@ml054

Reply:
oh yes - now I can remove those magic constants!